docs/tutorials/qml/ml_tools.md

```diff
@@ -94,6 +94,8 @@ config = TrainConfig(
 )
 ```
 
+If it is desired to only the save the "best" checkpoint, it must be ensured that (a) `checkpoint_best_only = True` is used while creating the configuration through `TrainConfig`, (b) `val_every` is set to a valid integer value (for example, `val_every = 10`) which controls the no. of iterations after which the validation data should be used to evaluate the model during training, which can also be set through `TrainConfig`, (c) a validation criterion is provided through the `validation_criterion`, set through `TrainConfig` to quantify the definition of "best", and (d) the dataloader passed to `train_grad` is of type `DictDataLoader`. In this case, it is expected that a validation dataloader is also provided along with the train dataloader since the validation data will be used to decide the "best" checkpoint. The dataloaders must be accessible with specific keys: "train" and "val". The criterion used to decide the "best" checkpoint can be customized by `validation_criterion`, which should be a function that can take any number of arguments and return a boolean value (True or False) indicating whether some validation metric is satisfied or not. Typical choices are to return True when the validation loss (accuracy) has decreased (increased) compared to smallest (largest) value from previous iterations at which a validation check was performed.
```

Comment:
Could you make the formatting a bit nicer? :)

E.g. maybe make the `(a), (b), (c)` points in a list of bulletpoints.

Reply:
done, resolving